Example/PaymentSheet Example/PaymentSheetUITest/EmbeddedUITest.swift

```diff
@@ -28,4 +28,177 @@ class EmbeddedUITests: PaymentSheetUITestCase {
+        try! fillCardData(app, cardNumber: "4000002500001001", postalEnabled: true)
+        app.buttons["Pay €50.99"].tap()
+        XCTAssertTrue(app.staticTexts["Success!"].waitForExistence(timeout: 10.0))
+
+        reload(app, settings: settings)
+
+        // Add Visa
+        app.buttons["Present PaymentSheet"].waitForExistenceAndTap()
+        app.buttons["+ Add"].waitForExistenceAndTap()
+        try! fillCardData(app, postalEnabled: true)
+        app.buttons["Pay €50.99"].tap()
+        XCTAssertTrue(app.staticTexts["Success!"].waitForExistence(timeout: 10.0))
+
+        // Switch to embedded mode kicks off a reload
+        app.buttons["embedded"].waitForExistenceAndTap(timeout: 5.0)
+        app.buttons["Present embedded payment element"].waitForExistenceAndTap()
+
+        XCTAssertTrue(app.buttons["•••• 4242"].waitForExistence(timeout: 3.0))
+        XCTAssertFalse(app.buttons["•••• 1001"].waitForExistence(timeout: 3.0))
+
+        // Switch from 4242 to 4444
+        app.buttons["View more"].waitForExistenceAndTap()
+        app.buttons["Edit"].waitForExistenceAndTap()
+        app.buttons["CircularButton.Edit"].waitForExistenceAndTap()
+        app.otherElements["Card Brand Dropdown"].waitForExistenceAndTap()
+        app.pickerWheels.firstMatch.swipeUp()
+        app.buttons["Done"].waitForExistenceAndTap()
+        app.buttons["Update"].waitForExistenceAndTap()
+
+        // Tap done on manage payment methods screen, then select 1010 card
+        app.buttons["Done"].waitForExistenceAndTap()
+        app.buttons["•••• 1001"].waitForExistenceAndTap()
+
+        XCTAssertTrue(app.buttons["•••• 1001"].waitForExistence(timeout: 3.0))
+        XCTAssertFalse(app.buttons["•••• 4242"].waitForExistence(timeout: 3.0))
+    }
+
+    func testMulipleCard_update_and_remove() {
+        var settings = PaymentSheetTestPlaygroundSettings.defaultValues()
+        settings.mode = .paymentWithSetup
+        settings.uiStyle = .paymentSheet
+        settings.customerKeyType = .legacy
+        settings.customerMode = .new
+        settings.merchantCountryCode = .US
+        settings.currency = .usd
+        settings.applePayEnabled = .on
+        settings.apmsEnabled = .off
+
+        loadPlayground(app, settings)
+
+        // Add MasterCard
+        app.buttons["Present PaymentSheet"].waitForExistenceAndTap()
+        try! fillCardData(app, cardNumber: "5555555555554444", postalEnabled: true)
+        app.buttons["Pay $50.99"].tap()
+        XCTAssertTrue(app.staticTexts["Success!"].waitForExistence(timeout: 10.0))
+
+        reload(app, settings: settings)
+
+        // Add visa
+        app.buttons["Present PaymentSheet"].waitForExistenceAndTap()
+        app.buttons["+ Add"].waitForExistenceAndTap()
+        try! fillCardData(app, postalEnabled: true)
+        app.buttons["Pay $50.99"].tap()
+        XCTAssertTrue(app.staticTexts["Success!"].waitForExistence(timeout: 10.0))
```

Comment:
Can we use the returning Customer here too? That way we can also test that the saved US Bank Account shows mandate when selected.